File: docs/examples/py/src/data_replication_remove.py
import asyncio

from reduct import Client, ReplicationSettings


async def main():
    # Create a client instance
    async with Client("http://127.0.0.1:8383", api_token="my-token") as client:
        # Remove the `example-replication` replication
        await client.delete_replication("example-replication")


loop = asyncio.get_event_loop()
loop.run_until_complete(main())

File: docs/examples/cpp/src/data_replication_remove.cc
#include <reduct/client.h>
#include <iostream>
#include <cassert>

using reduct::IBucket;
using reduct::IClient;
using reduct::Error;

int main() {
    // Create a client instance
    auto client = IClient::Build("http://127.0.0.1:8383", {
            .api_token = "my-token"
    });

    // Remove the `example-replication` replication
    auto err = client->RemoveReplication("example-replication");
    assert(err == Error::kOk);
}